helm/reference-wallet/templates/liquidity-deployment.yaml

```diff
@@ -64,6 +64,8 @@ spec:
           value: {{ .Values.sdk.faucet }}
         - name: CHAIN_ID
           value: {{ .Values.chainId | quote }}
+        - name: BLOCKCHAIN
```

Comment:
small additional change since first added blockchain var to liquidity

Reply:
You'll have to add `.Values.blockchain` (either a default or dummy value) to `helm/reference-wallet/values.yaml`, otherwise the template will fail to render.